go/api/go_api/session.py

```diff
@@ -12,7 +13,8 @@ class SessionStore(SessionBase):
     """
     def __init__(self, session_key=None):
         super(SessionStore, self).__init__(session_key)
-        self.session_manager = vumi_api().session_manager
+        self.session_manager = SessionManager(
+            get_redis_manager().sub_manager('session_manager'))
```

Comment:
The idea of the `VumiApi` is that it should create everything somewhat lazily, but I think putting a classmethod on `SessionManager` and `TokenManager` that encapsulates this setup sounds like a good compromise to me.